sections/compare-blender.liquid

```diff
@@ -1,17 +1,366 @@
+
+  var setCellHeight = function(obj, highest) {
+    for (var i in obj['childNodes']) {
+      if (obj['childNodes'][i].classList && obj['childNodes'][i].classList.contains('cell')) {
+        var height = highest.shift();
+        obj['childNodes'][i].style.height = height+'px';
+      }
+      if (obj['childNodes'][i].childNodes) {
+        setCellHeight(obj['childNodes'][i], highest);
+      }
+    }
+  };
+
+  var setCellHeightByGridItem = function (grid, highest) {
+    for (var i in grid['childNodes']) {
+      if (grid['childNodes'][i] && grid['childNodes'][i].classList && grid['childNodes'][i].classList.contains('grid__item')) {
+        var tempHighest = highest.slice();
+        setCellHeight(grid['childNodes'][i], tempHighest);
+      }
+    }
+  }
+
+  var getTallestHeight = function(grid) {
+    var highest = [];
+    for (var i in grid['childNodes']) {
+      if (grid['childNodes'][i] && grid['childNodes'][i].classList && grid['childNodes'][i].classList.contains('grid__item')) {
+        getHeightOfEachCell(grid['childNodes'][i], [], highest);
+      }
+    }
+    return highest;
+  };
+
+  var standardize_cell_size = function() {
+    var grids = document.getElementsByClassName('grid');
+    for (var i = 0; i < grids.length; i++) {
+      var arrOfHeights = [];
+      var highest = getTallestHeight(grids[i]);
+      setCellHeightByGridItem(grids[i], highest);
+    }
+  };
+
+  var makeOnlyLeftMostInfoVisible = function() {
+    var sections = document.getElementsByClassName('blender-info-side');
+    for (var i in sections) {
+      if (sections[i] && sections[i].style) {
+          sections[i].style.display = null;
+          var parentLeftDiff = sections[i].parentElement.offsetLeft;
+        if (sections[i].offsetLeft - parentLeftDiff > 25) {
+          sections[i].style.display = 'none';
+        } else {
+          sections[i].style.display = null;
+        }
+      }
+    }
+  };
+
+  var zoomin = function(product) {
+    var index = productSelectedVariant[product];
+
+    document.getElementById('popupBackground').style.display = null;
+    document.getElementById('thePopup').style.display = null;
+    document.getElementById('zoomInProduct').src = variantToOptions[index].src;
+    document.getElementById('zoomInPanel').src = variantToOptions[index].panel;
+  };
+
+  var exitPopup = function() {
+    document.getElementById('popupBackground').style.display = 'none';
+    document.getElementById('thePopup').style.display = 'none';
+  };
+
+standardize_cell_size();
+makeOnlyLeftMostInfoVisible();
+window.addEventListener("resize", function() {
+ standardize_cell_size();
+ makeOnlyLeftMostInfoVisible();
+});
+</script>
+
+
+<style>
+.product-comparison--bg-image{
+    background-image: url('{{ section.settings.top_image | img_url: '1000x' }}}');
+    background-position: 0 50%;
+    background-size: cover;
+    display: inline-block;
+    width: 100%;
+}
+.product-comparison--header-container {
+    padding: 100px 0 120px;
+}
```

Comment:
Can we reuse existing typography settings? Color size etc should probably just pick up site-wide styles.